src/http/one/MessageParser.js

```diff
@@ -84,8 +84,7 @@ export default class MessageParser {
 
         // replace obs-fold with a single space
         let rawH = raw;
-        // XXX: This does nothing (replace does not change string in place).
-        rawH.replace(/\r*\n\s+/, ' ');
+        rawH = rawH.replace(/\r*\n\s+/, ' ');
```

Comment:
I think this was meant to be `let rawH = raw.replace(/\r*\n\s+/, ' ');` instead.